samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -61,7 +61,7 @@ public DocumentTranslationClient(string endpoint, AzureKeyCredential credential,
         /// <summary> Returns the translation status for a specific document based on the request Id and document Id. </summary>
         /// <param name="id"> Format - uuid.  The batch id. </param>
         /// <param name="documentId"> Format - uuid.  The document id. </param>
-        /// <param name="options"> The request options. </param>
+        /// <param name="context"> The request options. </param>
```

Comment:
Would it be possible to update the comment here as well?

Reply:
Updated!